resources/js/profile-page/cover.tsx

```diff
@@ -126,9 +126,11 @@ export default class Cover extends React.Component<Props> {
     return (
       <>
         {this.props.user.is_supporter &&
-          <span className='profile-info__icon profile-info__icon--supporter' title={trans('users.show.is_supporter')}>
-            {times(this.props.user.support_level ?? 0, (i) => <span key={i} className='fas fa-heart' />)}
-          </span>
+          <a href={route('support-the-game')}>
```

Comment:
this breaks alignment, especially on mobile. The `span` should just be changed to `a` (and add the `href`) and have its style adjusted accordingly.